### PR TITLE
LPD-28956 Add additional configurations to email notifications when users are invited to work on a publication

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-28956

### CODE REVIEW

**Comment by brookedalton** [comment]
RE: original PR to BChan was closed with 
"_inviteUserToPublication pushes, and then tearDown pops it. That's a bit strange.

You should push the context and pop it from the same context (setUp/tearDown or within a test). But right now it's mixed.

So make a method called

_testGetInviteUsers() that pushes, does all the logic, and pops.

Have testGetInviteUsersEmailNotificationBody and the other testGetInviteUsers call _testGetInviteUsers but use Java functions to do the asserts.

Thx"

**Comment by brookedalton** [comment]
Hi @pei-jung, 
I moved the service context setup/teardown to `_testGetInviteUsers` but I'm not entirely sure I understood what was requested, please let me know if I need to make additional changes 🙇‍♀️

File: modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/portlet/action/test/InviteUsersMVCResourceCommandTest.java [hunk 1]
```diff
@@ -206,30 +195,31 @@ private ThemeDisplay _getThemeDisplay() throws Exception {
...
+		_serviceContext.setRequest(
 			mockLiferayResourceRequest.getHttpServletRequest());
 
-		ServiceContextThreadLocal.pushServiceContext(serviceContext);
+		ServiceContextThreadLocal.pushServiceContext(_serviceContext);
 
 		_mvcResourceCommand.serveResource(
 			mockLiferayResourceRequest, new MockLiferayResourceResponse());
+
+		ServiceContextThreadLocal.popServiceContext();
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/portlet/action/test/InviteUsersMVCResourceCommandTest.java]: Wrap this in a try-finally block to ensure that we always pop the service context even if unexpected exceptions occur in the resource command.

**Comment by brookedalton** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Comment by brookedalton** [comment]
ci:test:relevant

**Comment by brookedalton** [comment]
ci:forward

**Pull request closed without merging**